locales/en.default.json

```
@@ -164,6 +164,47 @@
         "sale_price": "Sale price",
         "unit_price": "Unit price"
       },
+      "unit_price": {
+        "per_item": {
+          "single": "{{price}} each",
+          "one": "{{price}} per {{count}} item",
+          "other": "{{price}} per {{count}} items"
+        },
+        "per_unit": {
+          "single": "{{price}}/{{unit}}",
+          "one": "{{price}}/{{unit}}",
```

You'd want to add `{{count}}` here, because the strings between `one` and `other` need to have the same interpolation strings.

```suggestion
          "one": "{{price}}/{{count}}{{unit}}",
```